readyplayerme/rpm-unreal-next-gen-sdk
# fix: updated preview method to load character asset correctly

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

-   fixed an issue that caused assets of the same type to stack after switching asset types. EG multiple pants being equipped 

![image](https://github.com/user-attachments/assets/9409f8c2-a9c3-44c5-bcab-69d49a80ae8d)







## How to Test

-   Test by running BasicLoader map. Equip an asset (EG top)
- then switch to another categoy and load an asset of that type (eg pants)
- then switch back to the first category and equip (top again)
- confirm that the old top is removed and new one is loaded 



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
